src/components/Events.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+import { getEvents } from '../redux/eventsReducer';
+import EventCat from './EventCat';
+
+const MainWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+
+  h2 {
+    color: #ffffff;
+    font-size: 2em;
+  }
+  a {
+    text-decoration: none;
+  }
+  input {
+    margin: 1em;
+    padding: 1em;
+    font-size: 2em;
+  }
+`;
+
+const ListWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 0.5em;
+  row-gap: 0.5em;
+`;
```

Comment:
- [ ] Your design still doesn't fit the mobile sizes. Let's improve it a bit so your design is responsive and fit the mobile screen sizes.
- Your design:
<img width="431" alt="image" src="https://user-images.githubusercontent.com/77190203/163080615-c361f5bb-f3f1-40e7-b798-6fa84e1ec408.png">

Reply:
<img width="1287" alt="Screenshot 2022-04-13 at 7 17 34 AM" src="https://user-images.githubusercontent.com/87834222/163086795-4f13beb5-5308-49f5-a812-41110b74f9d2.png">